ios/IJKMediaPlayer/IJKMediaPlayer/ijkmedia/ijkplayer/ios/pipeline/IJKVideoToolBox.m

```diff
@@ -1178,6 +1178,7 @@ static int vtbformat_init(VTBFormatDesc *fmt_desc, AVCodecParameters *codecpar)
     if (ret)
         goto fail;
     assert(context_vtb->fmt_desc.fmt_desc);
+    vtbformat_destroy(&context_vtb->fmt_desc);
```

Comment:
That's true, but it seems that the `fmt_desc` created here is just for testing whether `vtbformat_init` function works well. 
```
ret = vtbformat_init(&context_vtb->fmt_desc, context_vtb->codecpar);
if (ret)
    goto fail;
assert(context_vtb->fmt_desc.fmt_desc);
```
After that, a `vt_session`  is created by calling `vtbsession_create` function, and another `fmt_desc` is created during the creation of `vt_session`. And here comes the problem: the previous `fmt_desc` is lost. We didn't destroy it and nothing owns it now.

So we should free THIS `fmt_desc` before calling `vtbsession_create` function.